tests/unit/webpack.config.ts

```diff
@@ -125,4 +125,36 @@ describe('webpack.config.ts', () => {
 			});
 		});
 	});
+
+	describe('tslint', () => {
+		function getTslintLoader() {
+			const tsLintLoaders = config.module.rules.filter((rule) => rule.loader === 'tslint-loader');
+
+			return tsLintLoaders[0];
+		}
+
+		it('will cause build errors on linting warnings if not watching or testing', () => {
+			start(true, {});
+			const loader = getTslintLoader();
+			assert.isDefined(loader);
+			assert.strictEqual((<any> loader.options).emitErrors, true);
+			assert.strictEqual((<any> loader.options).failOnHint, true);
```

Comment:
In case you did not already know this, Chai also has an `isTrue` method.

Reply:
Changed :)